MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -0,0 +1,229 @@
+    private RecyclerView mRecyclerView;
+    private DetailAdapter mAdapter;
+    private List<DetailModel> mData;
+    private Object Utils;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        mRecyclerView = findViewById(R.id.recycler_view);
+        mRecyclerView.setLayoutManager(new LinearLayoutManager(this));
+        mData = addItemsFromJSON();
+        mAdapter = new DetailAdapter(this);
+        mAdapter.setList(mData);
+        mRecyclerView.setAdapter(mAdapter);
+        mRecyclerView.addItemDecoration(new DividerItemDecoration(this, DividerItemDecoration.VERTICAL));
+    }
+
+    private List<DetailModel> addItemsFromJSON() {
+        final List<DetailModel> list = new ArrayList<>();
+        try {
+
+            String jsonDataString = readJSONDataFromFile();
+            JSONArray jsonArray = new JSONArray(jsonDataString);
+
+            for (int i = 1; i < jsonArray.length(); ++i) {
+                JSONObject itemObj = jsonArray.getJSONObject(i);
+                final int bool;
+                if(i%3==1)bool=0;
+                else bool=1;
+                String name = itemObj.getString("name");
+                String login = itemObj.getString("login");
+                String location = itemObj.getString("location");
+                int contributions = itemObj.getInt("contributions");
+                int followers = itemObj.getInt("followers");
+                String organizations = itemObj.getString("organizations");
+                list.add(new DetailModel(name,login,location,contributions,followers,organizations,bool));
+                Log.d("data###", location);
+            }
+          return list;
+
+        } catch (IOException | JSONException e) {
+            Log.d("addItemsFromJSON: ", String.valueOf(e));
+        }
+        return list;
+    }
+
+    private String readJSONDataFromFile() throws IOException{
+        InputStream inputStream = null;
+        StringBuilder builder = new StringBuilder();
+
+        try {
+
+            String jsonString = null;
+            inputStream = getResources().openRawResource(R.raw.github_users);
+            BufferedReader bufferedReader = new BufferedReader(
+                    new InputStreamReader(inputStream, "UTF-8"));
+
+            while ((jsonString = bufferedReader.readLine()) != null) {
+                builder.append(jsonString);
+            }
+
+        } finally {
+            if (inputStream != null) {
+                inputStream.close();
+            }
+        }
+        return new String(builder);
+    }
+
+    public void onItemClicked(int position) {
+        final DetailModel model = mData.get(position);
+        Toast.makeText(MainActivity.this, "Followers: " + model.getFollowers() + ", Contributions: " + model.getContributions(), Toast.LENGTH_SHORT).show();
```

Comment:
Toast should be displayed in the format `Type#a/b: followers: <number>, contributions: <number>` based on cell type.